.gitattributes

```diff
@@ -1,4 +1,51 @@
-# Ignoring files for distribution archieves
+# Define the line ending behavior of the different file extensions
+# Set default behaviour, in case users don't have core.autocrlf set.
+* text=auto
+* text eol=lf
+
+# Explicitly declare text files we want to always be normalized and converted
+# to native line endings on checkout.
+*.php text
+*.default text
+*.ctp text
+*.md text
+*.po text
+*.js text
+*.css text
+*.ini text
+*.txt text
+*.xml text
+
+# Declare files that will always have CRLF line endings on checkout.
+*.bat eol=crlf
+
+# Declare files that will always have LF line endings on checkout.
+*.pem eol=lf
+
+# Denote all files that are truly binary and should not be modified.
+*.png binary
+*.jpg binary
+*.gif binary
+*.ico binary
+*.mo binary
+
+# Remove files for archives generated using `git archive`
+appveyor.yml export-ignore
+CONTRIBUTING.md export-ignore
+.editorconfig export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+Makefile export-ignore
+phpunit.xml.dist export-ignore
+.travis.yml export-ignore
+.scrutinizer.yml export-ignore
+.stickler.yml export-ignore
+tests/test_app export-ignore
+tests/TestCase export-ignore
+.github export-ignore
+
+
+# Ignoring files for distribution archives
```

Comment:
Here are your old settings, above the one from cakephp/cakephp - I didn't feel versed to make the decisions/merge.

Reply:
Can you take a look at these specifically and (tell me what to) remove what isn't required anymore? Should I just remove the dupes and we are ok?

Reply:
Been thinking about this and I would prefer to list specific files for the package.

Reply:
While I understand what this feature does https://feeding.cloud.geek.nz/posts/excluding-files-from-git-archive/ I don't know why I would want to exclude anything. So can you list what you would want to exclude to make it easy?

Reply:
Everything besides:
* `src`
* `composer.json`
* `composer.lock`
* `README.md`
* `LICENSE`